### DISCUSSION TITLE
Mass balance issues

### DISCUSSION THREAD

**Discussion by safoah** [metrumresearchgroup/mrgsolve#791] [Q&A]
I am a building a PBPK model and to verify it I added all the compartment to check for mass balance. My mass decrease over time, I added a clearance compartment but I am stilling having the same issue with my simulations. Any suggestions?

**Comment by kylebaron** [comment]
Hi @Nbamfo; I moved your issue over to the discussions section; you can find it here:
https://github.com/metrumresearchgroup/mrgsolve/discussions/

I think the only thing you can do is look at the math and see where mass isn't balancing.  It might help to simplify the model back to the point where mass balance is maintained and then add back complexity.  I'm not sure how much is going on in your model but if there is a lot of complexity, that approach may help.

**Reply by safoah** [reply]
Thank you @kylebaron, that was a helpful suggestion.
